docs/docs/examples/cookbooks/llama3_cookbook_ollama_replicate.ipynb

```diff
@@ -0,0 +1,1334 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    resp = llm.stream_complete("do you like drake or kendrick better?")
</pre><p>spicy, i was thinking to do something similar in my RBC talk... but I asked the LLM to write something in the style of Drake... was thinking to do something in the style of Kdot as well.</p><p> </p><p>ohhhh i read a bit further. I see you did this :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>